### PR TITLE
Welcoming contact field

### PR DESCRIPTION
**Pull request by comradekingu**
Want to contribute to the translations. :)

### CODE REVIEW

**Comment by TurnrDev** [comment]
Hi @comradekingu, translations to the bot can be provided via [weblate](https://hosted.weblate.org/projects/trainerdex/discord-bot/)

**Comment by comradekingu** [comment]
@TurnrDev The point is I can't do so in good conscience when you have that in the way.

**Comment by TurnrDev** [comment]
The Code of Conduct basically says Don't Be a Dick? I don't see how it affects anything, unless you plan on being a dick?

**Comment by comradekingu** [comment]
@TurnrDev
If I am correct in assuming you haven't read it, don't remember what it actually said, asked nobody about including it, and followed up nowhere to ensure people actually did so voluntarily, it is like every other project; a green light in a checklist. I don't deny you the right to impose it, or break it for that matter, but it means detailing my concerns is as far as we get towards something more productive.
To borrow a bit from an e-mail I sent recently.
It affects my conscience of wanting to contribute to a libre software projects, under terms that can be attributed to it as such.
My problem it is that of invoking immutable characteristics for readers of said document, for every project they contribute to. That is not warranted.
That is of course only for the people that do, which isn't to say that those antisocial enough to do anything warranting mention have read it.
There is a clear mismatch between who would stand to benefit from it, and who those people are, and a further mismatch between the people that do, believing of others that they have.

In the hypothetical scenario that antisocial individuals would, the idea hinges on more concern
for a text-document than actually carrying out the behavior.
So where it would otherwise be justifiable by said people, now there is a text-document preventing what? Is a document needed to point out antisocial behavior? That doesn't hold.
It isn't just bloat though.

A far more, and likely and unfortunately reality, is that antisocial people can actually
gauge not only what is within the limit of offenses, but actually use the code against others
in a malicious way.

Moreover, later versions of what is continually growing in scope has "project maintainers"
replaced with "community leaders"…
People with less vested interest can thereby use what possibility of censorship is brought to bare in the text. It is total, and retroactive.

Its expressed intent is to thwart meritocracy, which I find disagreeable.
This includes a ban on politics, sexuality, a misunderstanding of what empathy is, and
a misuse of the word and legal concept of _harassment_.
It is obviously impossible to forego intent in any practical application,
but where its relevance vanishing into nothing seems good on paper, is precisely where it opens up the worst type of authoritarianism.

Not that I think many project, or DiscordBot in particular would fall victim to it in its fullest extent, but I don't want to pledge allegiance to it, and certainly don't want to have my effort implied as somehow aligned.

It is impossible to put this kind of thing in a libre license, but it didn't keep the author from trying,
as now there is a non-free licensing scheme to that effect.

TL;DR I favour community over "professional setting".

**Comment by TurnrDev** [comment]
> If I am correct in assuming you haven't read it, don't remember what it actually said, asked nobody about including it, and followed up nowhere to ensure people actually did so voluntarily

I don't know what you're going on about man. I'm the sole maintainer of a personal project. It's literally just a post asking people not to discriminate on others if they come in here. Nothing more, nothing less. The fact that you have an issue with that raises a few eyebrows. 🤨 

> A far more, and likely and unfortunately reality, is that antisocial people can actually
gauge not only what is within the limit of offenses, but actually use the code against others
in a malicious way.

Well, no, because if someone is a dick, they're going to be banned from contributing (if GitHub even allows that, I don't even know.) - I'm not going to turn around and be like "Oh, what you said was technically not listed as a **don't** on the Code of Conduct so you're good" because the spirit of a document applies - "Don't be a dick."


> This includes a ban on politics

Yes, this is an OSS, not a local Facebook group. Keep politics out of here. 

> Not that I think many project, or DiscordBot in particular would fall victim to it in its fullest extent, but I don't want to pledge allegiance to it, and certainly don't want to have my effort implied as somehow aligned.

I honestly don't see the problem. If you want to contribute translations to the project, you'd be doing do off-site on weblate instead of GitHub. You don't need to interact with anybody else. You're well within your right to not to reply to someone you don't agree with. Not that you'll come across many people. TrainerDex has 12k users, 1 developer, and a small handful of translators (most of which are users of the platform themselves).

**Comment by comradekingu** [comment]
I am my own personal conspiracy.
Your description of what the text says isn't accurate. It actually starts off discriminating people for reasons beyond their control from others. That is not a positive psychological feature.
What discrimination is must IMO rely on some objective reality. If it is all down to "experience", then intent doesn't matter. We are no longer talking about discrimination or harassment in the traditional sense there, but rather a made up concept that isn't governable in anything other than rank authoritarianism. Nobody is made a better person that way.

Banning "The use of sexualized language" is a type of censorship, and your earlier post runs afoul of it. This is where I agree with you, and I would like to be on equal footing. I agree that what the text actually says is somewhat hard to remember.
Before we are done banning politics, we are here discussing politics. I agree with you again, get rid of it. It then gets to "Other conduct which could reasonably be considered inappropriate in a professional setting". This is not community values for a community. Those are corporate values for corporate interest.

We would all know what the specifics of the GitHub counterpart would be, if we read it.
I have, and I can't remember exactly what it says.
So if it is redundant to another textfile, and most importantly law, why is it there?
The author states it is a political tool.
It does otherwise apply to translations "This Code of Conduct applies both within project spaces …"
That spreads false confidence for people thinking it does anything desireable over the proposed alternative. I am planning to be nothing other than on my best behavior, and am offering to spend my free time.
In a personal capacity I don't like just not contributing without trying to make amends, and I am intrigued as to how this way of policing communities made its way into some libre software communities despite not gaining any popularity.

**Comment by TurnrDev** [comment]
If you decide to contribute, or not, is no skin of my back. But I'm not removing the current Code of Conduct unless someone has a valid alternative suggestion. A two line "sort it out among yourselves" message is insufficient.

**Comment by comradekingu** [comment]
@TurnrDev Why, and for whom?

**Comment by TurnrDev** [comment]
Insufficient because it's too vague. I'm locking this thread now. This conversation is not a productive use of my time. This repo isn't even maintained except bug fixes.
